lepton/ldd_linux.go

```diff
@@ -69,6 +82,9 @@ func getSharedLibs(targetRoot string, path string) ([]string, error) {
 	if err == nil && IsDynamicLinked(elfFile) {
 		env := os.Environ()
 		env = append(env, "LD_TRACE_LOADED_OBJECTS=1")
+		if targetRoot != "" {
+			env = append(env, "LD_LIBRARY_PATH="+targetRoot)
+		}
```

Comment:
actually this might be what the problem is - this won't work on mac - i realize this file is for linux but will need something corresponding for ldd_darwin.go

Reply:
I modified findLib() in ldd_darwin.go to take into account the target root, and now it works on mac too.